packages/app-desktop/InteropServiceHelper.ts

```diff
@@ -78,7 +78,9 @@ export default class InteropServiceHelper {
 					shim.setTimeout(async () => {
 						if (target === 'pdf') {
 							try {
+								win.webContents.executeJavaScript('document.querySelectorAll(\'details\').forEach(el=>el.setAttribute(\'open\',\'\'))');
 								const data = await win.webContents.printToPDF(options);
+								win.webContents.executeJavaScript('document.querySelectorAll(\'details\').forEach(el=>el.removeAttribute(\'open\',\'\'))');
```

Comment:
What if some of the `details` elements had `open="true"` before line 81? It might make more sense to only close the `details` elements that were previously `open`.

Reply:
@personalizedrefrigerator I just tested on joplin v2.9.17.
Even if you keep some `details` tags open, whenever you quit joplin and start again, it will go back to the closed state. They don't survive restarts. 

> It might make more sense to only close the details elements that were previously open.

This can be done, but the question is, do we want to do it? Because at some point the tags are gonna close by themselves.

Reply:
I was thinking the same thing, but then I realize we're loading a temporary HTML file when printing so it's ok to make change to the DOM (since the changes won't be applied to the note).

Reply:
In fact, why do you close the "details" element after printing?

Reply:
Now that you mention it, closing them is not necessary at all. Should I remove the closing part?